pkg/mcproducer/scripts/replace-kernel-module.sh

```diff
@@ -19,17 +21,22 @@ EOF
     cat "$kmm_config_file_filepath"
 }
 
-echo "before checking image tar file presence"
-if [ -e $kernel_module_image_filepath ]; then
-    echo "Image file $kernel_module_image_filepath found on the local file system, creating kmm config file"
+echo "before checking image presence"
+if [ -n "$(podman images -q $worker_image 2> /dev/null)" ]; then
```

Comment:
the same here - we should check the second image to be pulled by the pull script